### ISSUE TITLE
Probably add link to co2wui to open the console? 

### ISSUE THREAD

**Issue by dimitriskomnos** [JRCSTU/co2wui#116]
I think it would be nice, 
not very urgent thought. 

@ankostis what do you think?

**Comment by ankostis** [comment]
Yes, that would be handy for the users, but needs some research to work correctly when multiple, installation exist in parallel..

But do users still need the console for any task?
